src/test_workflow/integ_test/integ_test_suite_opensearch.py

```diff
@@ -72,19 +73,26 @@ def __setup_cluster_and_execute_test_config(self, config: str) -> int:
         if "additional-cluster-configs" in self.test_config.integ_test.keys():
             self.additional_cluster_config = self.test_config.integ_test.get("additional-cluster-configs")
             logging.info(f"Additional config found: {self.additional_cluster_config}")
-        with LocalTestCluster.create(
+        if self.additional_cluster_config is None:
+            self.additional_cluster_config = {"cluster.name": "opensearch1"}
+        if "topology" in self.test_config.integ_test.keys():
+            self.topology = self.test_config.integ_test.get("topology")
+        else:
+            self.topology = [{'cluster_name': 'cluster1', 'data_nodes': 1, 'master_nodes': 0}]
```

Comment:
The `self.topology` property should return an instance of a class similar to `InputManifest.*` classes and all JSON manipulation self-contained in there. For example https://github.com/opensearch-project/opensearch-build/blob/main/src/manifests/input_manifest.py#L126.